pymc3/stats.py

```diff
@@ -104,7 +103,7 @@ def autocov(x, lag=1):
 
 def dic(trace, model=None):
     """Calculate the deviance information criterion of the samples in trace from model
-    Read more theory here - in a paper by some of the leading authorities on model selection -
+    Read more theory here - in a paper by some of the leading authorities on Model Selection -
```

Comment:
Could you please change this back (also some of the changes below)? Some of the change in this file is related to https://github.com/pymc-devs/pymc3/pull/2606, and you are changing them back here (accident during merging master to your branch?). If it helps you can also copy from master and just reimplement you change.